File: Java/Logica_Exercicios/Repeticao/RepeticaoDoWhile.java
import java.util.Scanner;

public class RepeticaoDoWhile {
    public static void main(String[] args)  {
    	Scanner ler = new Scanner(System.in);
		char tipoCurso = ' ';

		do {   
			System.out.println("Tipo do Curso [G/P]: ");
			tipoCurso = ler.nextLine().charAt(0);
			tipoCurso = Character.toUpperCase(tipoCurso);

		    if (tipoCurso!='G' && tipoCurso!='P') {
		      System.out.println("Tipo de curso incorreto\n");
		    }
		} while (tipoCurso !='G' && tipoCurso !='P' );

		if (tipoCurso=='G'){
		   System.out.println("Graduacao");
		}
		else {
		   System.out.println("Pos-graducao");
		}
    } 
}

File: Js/Condicional/repeticaoDoWhile.js
	    var tipoCurso;
	    do {	
		    tipoCurso = prompt("Tipo do Curso [G/P]: ");
    	    tipoCurso = tipoCurso.toUpperCase();

		    if (tipoCurso!='G' && tipoCurso!='P') {
		      console.log("Tipo de curso incorreto");
		    }
		} while (tipoCurso !='G' && tipoCurso !='P' );

		if (tipoCurso=='G'){
		   console.log("Graduacao");
		}
		else {
		   console.log("Pos-graducao");
		}

